esphome/components/mitsubishi_itp/mitsubishi_uart-climatecall.cpp

```diff
@@ -78,6 +78,27 @@ void MitsubishiUART::control(const climate::ClimateCall &call) {
   if (call.get_target_temperature().has_value()) {
     target_temperature = call.get_target_temperature().value();
     set_request_packet.set_target_temperature(call.get_target_temperature().value());
+
+    // update our MHK tracking setpoints accordingly
+    switch (mode) {
+      case climate::CLIMATE_MODE_COOL:
+      case climate::CLIMATE_MODE_DRY:
+        this->last_cool_setpoint_ = target_temperature;
+        break;
+      case climate::CLIMATE_MODE_HEAT:
+        this->last_heat_setpoint_ = target_temperature;
+        break;
+      case climate::CLIMATE_MODE_HEAT_COOL:
+        if (this->get_traits().get_supports_two_point_target_temperature()) {
+          this->last_heat_setpoint_ = target_temperature_low;
+          this->last_cool_setpoint_ = target_temperature_high;
+        } else {
+          // this->last_heat_setpoint_ = target_temperature;
+          // this->last_cool_setpoint_ = target_temperature;
+        }
```

Comment:
A little confused about this one.  If we get a HEAT_COOL call, but don't support dual points, shouldn't we still set the last_*_setpoint to the set value?  (Genuine question, I'm not sure)

Reply:
This is... tricky, since the MHK only ever has the two setpoints. If we set both to the target temperature (which we definitely can do), we'd end up clobbering both the heat and cool setting displayed on the MHK itself.

This causes a problem as the MHK *needs* those two setpoints to be different to decide what temperature to set. Note that this is also part of a (somewhat) unsolved problem currently. The MHK at present will "take over" an auto mode issued by the ESP and cause all sorts of headaches. Interestingly, my observation matches that setting ACO on the remote does not affect the MHK either (and the MHK steals control as a result).